src/app/core/user-module/user.service.ts

```diff
@@ -95,6 +75,16 @@ export class UserService {
         );
   }
 
+  getUsersLearningObjects(username: string): Promise<LearningObject[]> {
+    return this.http
+      .get(USER_ROUTES.GET_USER(username), { withCredentials: true })
```

Comment:
Nit: Can we update one of the names either the function or the route name. because one looks like we are getting all learning objects for a user and the other looks like we are getting the user's information.

Reply:
I just looked at the route in the USER_ROUTES. that is the route to get the user's document in the users collection not the route to get learning objects for a user.

Reply:
changed to use appropriate route =)